apps/test/unit/templates/progress/ProgressBubbleTest.js

```diff
@@ -288,6 +288,19 @@ describe('ProgressBubble', () => {
       assert.include(wrapper.find('a').prop('href'), 'section_id=12345');
     });
 
+    it('includes the user_id in the queryparams if selectedStudentId is present', () => {
+      fakeLocation.href = "http://studio.code.org/s/csd3/stage/3/puzzle/7";
+      const wrapper = shallow(
+        <ProgressBubble
+          {...defaultProps}
+          currentLocation={fakeLocation}
+          selectedSectionId="12345"
+          selectedStudentId="207"
+        />
+      );
+      assert.include(wrapper.find('a').prop('href'), 'user_id=207');
+    });
+
```

Comment:
`ProgressBubble` is already fairly well tested, and I added in a quick check that the param for the user_id is making it into the href as expected.